docker/Dockerfile-17.06.1

```diff
@@ -1,4 +1,4 @@
-FROM launcher.gcr.io/google/ubuntu16_04
+FROM gcr.io/cloud-builders/docker-base:latest
```

Comment:
These should be `gcr.io/$PROJECT_ID/docker-base:latest` -- the intention is that any user can run these builds in their own project to get a `docker` builder in their own registry.

Does the name of the base image matter at all though? Could it just be `docker-base` since it's not pushed?

Reply:
Done